src/vivarium_public_health/population/add_new_birth_cohorts.py

```diff
@@ -168,17 +109,13 @@ class FertilityAgeSpecificRates:
     def setup(self, builder):
         """ Setup the common randomness stream and
         age-specific fertility lookup tables.
-
         Parameters
         ----------
         builder : vivarium.engine.Builder
             Framework coordination object.
-
         """
-
         self.randomness = builder.randomness.get_stream('fertility')
-        asfr_data = builder.data.load("covariate.age_specific_fertility_rate.estimate",
-                                      future=builder.configuration.input_data.forecast)
+        asfr_data = builder.data.load("covariate.age_specific_fertility_rate.estimate")
```

Comment:
if this is old code, I think we want to keep the future flag or building artifacts for diarrhea won't use the forecast asfr data

Reply:
Good catch